mreg_cli/config.py

```diff
@@ -91,10 +86,12 @@ def _load_env_config() -> dict[str, str]:
         }
 
     @overload
-    def get(self, key: str) -> str | None: ...
+    def get(self, key: str) -> str | None:
+        ...
 
     @overload
-    def get(self, key: str, default: DefaultType = ...) -> str | DefaultType: ...
+    def get(self, key: str, default: DefaultType = ...) -> str | DefaultType:
+        ...
```

Comment:
Can you investigate what causes this? My Ruff v0.4.6 formats stub functions on a single line.

Reply:
I did try. And I failed. I think it's VSCode somewhere. `tox -e lint-fix` also doesn't reformat this. We should probably have a pre-commit hook to do this.